minecraft/common/src/main/java/com/zigythebird/playeranim/animation/PlayerAnimationController.java

```diff
@@ -28,6 +32,9 @@ public class PlayerAnimationController extends AnimationController {
             "body", new Vec3f(0, 12, 0)
     );
 
+    //Used for applying torso bend to bones like the head.
+    protected List<AdvancedPlayerAnimBone> top_bones;
```

Comment:
I absolutely hate the amount of shitty code you’re writing just to fix the torso bend